recipes-core/images/avnet-image-minimal.bb

```diff
@@ -2,3 +2,25 @@ DESCRIPTION = "OSL image definition for Avnet boards"
 LICENSE = "MIT"
 
 require avnet-image-minimal.inc
+
+# remove unnecessary packages found in
+# ${IMAGE_INSTALL:pn-petalinux-image-minimal} (build/conf/plnxtool.conf)
+# that make the OS image too large to fit in QSPI
+# Due to Yocto remove syntax having highest priority, removing within
+# avnet-image-minimal.inc prevents these packages being included in avnet-image-full
+
+IMAGE_INSTALL:remove:zynq = "\
+        bridge-utils \
+        can-utils \
+        htop \
+        meson \
+        openssh-sftp-server \
```

Comment:
Ohhh this seems kinda messy to remove (default) packages in one file (.bb file) while adding other desired packages in another file (.inc file).  This yocto mechanism for rootfs config has always been confusing and fraught with peril.  I know I have messed it up over the years.  Is there a different (and hopefully better) way to do this rootfs config in general?